pkg/bench/summary.go

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package bench
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cilium/tetragon/pkg/metrics/ringbufmetrics"
+	"github.com/fatih/color"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// Summary gathers benchmark results. Serializes to JSON.
+// This is updated from multiple places concurrently, but currently
+// there is no overlap on writes, so this isn't yet protected by a mutex.
```

Comment:
Are the concurrent writes all happen in BpfStats?

Reply:
it's the ExportStats that are updated from tetragon process,
rest should be done just in bench process, I'll update the comment